### PR TITLE
debug infinite scroll and add styling for new message format  

### PR DESCRIPTION
**Pull request by leeannachen**

### CODE REVIEW

File: proj/src/main/webapp/style.css [hunk 1]
```diff
@@ -251,23 +266,23 @@ footer a:hover {
...
+  width: 12%;
+  height: 3vh;
 
   background: #4F4F4F;
   box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
   border-radius: 32px;
   border-width: 0;
 
-  font-size:2vmin;
+  font-size:2rem;
```
Comment (aokeke) [inline proj/src/main/webapp/style.css:285]: space after the colon

File: proj/src/main/webapp/style.css [hunk 2]
```diff
@@ -213,35 +217,46 @@ footer a:hover {
...
-  padding: 25px;
-  flex-grow: 1;
-  font-size:1.5rem;
+
+.message-body {
+  font-family: 'Montserrat', sans-serif;  
+  font-size: 1.5rem;
+  background: none;
+  border: none;
+  padding:0;
```
Comment (aokeke) [inline proj/src/main/webapp/style.css]: space after the colon

**Review by aokeke — APPROVED** [review]
CSS witchcraft! LGTM

File: proj/src/main/webapp/style.css (hunk 1 above)
Comment (leeannachen) [reply proj/src/main/webapp/style.css:285]: fixed

File: proj/src/main/webapp/style.css (hunk 2 above)
Comment (leeannachen) [reply proj/src/main/webapp/style.css]: fixed

**Review by nlumiere — APPROVED** [review]
I doubt anything breaks if this goes through :)

**Review by showermat — APPROVED** [review]

**Pull request merged**